src/components/LaunchFilter/components/FilterLink/FilterLink.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../../../shared/styles/variables';
+
+.launches-filter-item {
+  display: flex;
+  width: 5rem;
+  height: 5rem;
+  
+  background-color: $primary-main;
+  border-radius: 50%;
+
+  &:not(:last-child) {
+    margin-bottom: 1rem;
+  }  
+
+  &:hover,
+  &:focus {
+    width: 16rem;
+    border-radius: 50% / 7.5rem;
+    background-color: #ffee58; //var
```

Comment:
1. Do we need division?
2. You are right. Use var :)

Reply:
1. It's not division, it's one of the syntax variants
2. I'll change to var after Sasha's PR merging

Reply:
2. Changed to var